examples/apollo-cache/cache-control.ts

```diff
@@ -3,12 +3,12 @@ import { Directive } from "type-graphql";
 import { type RequireAtLeastOne } from "./helpers/RequireAtLeastOne";
 
 export function CacheControl({ maxAge, scope }: RequireAtLeastOne<CacheHint>) {
-  if (!maxAge && !scope) {
+  if (maxAge === undefined && !scope) {
```

Comment:
I'm not sure if this whole check is needed because types already have the `RequireAtLeastOne`.